static-assets/components/cstudio-common/common-api.js

```diff
@@ -667,6 +667,7 @@ var nodeOpen = false;
 
                         this.on("submitComplete", function(evt, args){
                             //window.location.reload();
+                            CStudioAuthoring.SelectedContent.selectedContent = []; // clear selected contents after publish
```

Comment:
This really should be common method clear or clean up stuff after actions.
Also CStudioAuthoring.SelectedContent should have a clear method... this code operates directly on the datastructure